x-pack/plugins/secops/public/components/timeline/body/renderers/get_column_renderer.test.tsx

```diff
@@ -6,32 +6,41 @@
 
 import { mount, shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
-import { cloneDeep, get } from 'lodash/fp';
+import { cloneDeep } from 'lodash/fp';
 import * as React from 'react';
 
-import { Ecs } from '../../../../graphql/types';
-import { mockEcsData } from '../../../../mock';
+import { TimelineNonEcsData } from '../../../../graphql/types';
+import { mockTimelineData } from '../../../../mock';
 import { TestProviders } from '../../../../mock/test_providers';
 import { getEmptyValue } from '../../../empty_value';
 import { defaultHeaders } from '../column_headers/default_headers';
 
 import { columnRenderers } from '.';
 import { getColumnRenderer } from './get_column_renderer';
+import { deleteItemIdx, findItem } from './helpers';
 
-describe('get_column_renderer', () => {
-  let nonSuricata: Ecs;
+const getValues = (field: string, data: TimelineNonEcsData[]): string[] | undefined => {
```

Comment:
May want to extract this as a helper as it's duplicated across:
* `plain_column_renderer.test.tsx`
* `unknown_column_renderer.test.tsx`
* `get_column_renderer.test.tsx`

Reply:
funny I thought that I did it, anyway thank you